api/translate/translate/v1/translate.proto

```diff
@@ -1,5 +1,16 @@
 syntax = "proto3";
 package translate.v1;
+option go_package = "./gen";
 
-service TranslateService {
+service TranslateService {}
```

Comment:
I think it's better to move service at the end, before messages, requests, and responses. But that just styling. Also, files should end with a newline as far as I know.